frontend/src/components/common/form/ActionButton.tsx

```diff
@@ -50,15 +72,19 @@ const ActionButton = ({
     controlledSetLoading?.(true);
     try {
       await onClick();
-      if (showDefaultToasts) {
+      if (showDefaultToasts || messageOnSuccess) {
```

Comment:
is there a way to not have all these flags (showDefaultToasts, messageOnSuccess, generateErrorMessage)